.vsts-ci/templates/test/nix-test-steps.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  purpose: ''
+  tagSet: 'CI'
+
+steps:
+  - pwsh: |
+      Get-ChildItem -Path env:
+    displayName: Capture Environment
+    condition: succeededOrFailed()
+
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download build artifacts'
+    inputs:
+      downloadType: specific
+      itemPattern: |
+        build/**/*
+      downloadPath: '$(System.ArtifactsDirectory)'
+
+  - pwsh: |
+      Get-ChildItem "$(System.ArtifactsDirectory)\*" -Recurse
+    displayName: 'Capture Artifacts Directory'
+    continueOnError: true
+
+  - pwsh: |
+      if (Get-Command tdnf -errorAction SilentlyContinue) {
```

Comment:
is tdnf available for all *nix, or do we worry about this later?

Reply:
This step will be removed before merging.  These packages have been added to all Linux images.